test/code/providers/os_provider/osprovider_test.cpp

```diff
@@ -306,14 +317,20 @@ class OSProvider_Test : public CPPUNIT_NS::TestFixture
 #if !defined(PF_DISTRO_ULINUX)
         CPPUNIT_ASSERT(6 <= strCaption.length());
 #endif
+
         CPPUNIT_ASSERT(uiCountProcesses > 0);
-        CPPUNIT_ASSERT(256000 <= ulTotalSwap);
-        CPPUNIT_ASSERT(512000 <= ulTotalVM);
-        CPPUNIT_ASSERT(ulFreeVM <= ulTotalVM);
+
+        if ( ! fRunningOnTravis )
```

Comment:
In a further checkin ... already changed on disk. Want separate commits, this is not the head.